R/transformFunctions.R

```diff
@@ -72,3 +72,9 @@ YeoJohnson <- function(x, lambda) {
 
   return(result)
 }
+
+#' @export
+logit <- function(x) { stats::qlogis(x) }
+
+#' @export
+invLogit <- function(x) { stats::plogis(x) }
```

Comment:
how about
```suggestion
#' @export
logit <- stats::qlogis

#' @export
invLogit <- stats::plogis
```
?

Reply:
that would work fine of course, but it would also expose additional arguments of the `*logis` functions, which may be confusing when we make these functions available in the syntax mode